docs/source/tutorials/first_kagome_chain.md

```diff
@@ -62,12 +66,21 @@ subkey inspect //Bob
 For this tutorial you can start a single node network as follows:
 
 ```
-kagome_validating \
-    --genesis config/localchain.json \
-    --keystore config/localkeystore.json \
-    --leveldb ldb \
-    --rpc_http_port 40363 \
-    --rpc_ws_port 40364
+docker run \
+  -it \
+  -v $(pwd)/demo:/kagome/configs \
+  -p 30363:30363 \
+  -p 9933:9933 \
+  -p 9944:9944 \
+  soramitsu/kagome \
+    kagome_validating \
+      --genesis config/localchain.json \
+      --keystore config/localkeystore.json \
+      --leveldb ldb \ 
+      --p2p_port 30363 \
+      --rpc_http_port 9933 \
+      --rpc_ws_port 9944 \
+      -f
```

Comment:
I believe this flag is not covered in the description below.

Reply:
It seems more clear to write the full option here ```--finalizing_only``` or smth like this.

Reply:
Or this, right.